cli/azd/internal/repository/app_init.go

```diff
@@ -154,6 +154,26 @@ func (i *Initializer) InitFromApp(
 						prj.AzureDeps[depIndex] = eventHubs
 					}
 				}
+				if eventHubs, ok := dep.(appdetect.AzureDepEventHubs); ok && !eventHubs.UseKafka {
+					for key, destination := range prj.Metadata.BindingDestinationInProperty {
+						if destination == "" {
+							err := promptMissingPropertyAndExit(i.console, ctx, key)
+							if err != nil {
+								return err
```

Comment:
Are you sure `return err` will exit azd program?

Reply:
Align with the previous practices, we exit it directly, instead of returning error.